tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Azure/Functions/IDelayedException.cs

```diff
@@ -0,0 +1,18 @@
+// <copyright file="IDelayedException.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if !NETFRAMEWORK
+
+using System;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Azure.Functions
+{
+    internal interface IDelayedException
+    {
+        public Exception Exception { get; }
+    }
+}
```

Comment:
We said `#nullable enable` for new files ideally

```suggestion
#if !NETFRAMEWORK

#nullable enable

using System;

namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Azure.Functions
{
    internal interface IDelayedException
    {
        public Exception? Exception { get; }
    }
}
```

Reply:
Sorry, I didn't get the memo 😛